### DISCUSSION TITLE
Checking Device Configuration Requirement

### DISCUSSION THREAD

**Discussion by Vedsaga** [Kong/insomnia#3981] [Help] [answered]
I eveyone, I just wanted to know where could I see if the version which I am downloading is compatible to my device or not? Currently, I am using `windows 7 32bit` and having hard time finding the exact version which I can download.

**Comment by dimitropoulos** [comment, marked as answer]
check out the [FAQ](https://support.insomnia.rest/article/155-frequently-asked-questions) on the docs.

> Does Insomnia provide 32-bit binaries?
> At this time, Insomnia is 64-bit only.
